### PR TITLE
Fixed windows fetching & rolled Chromium to r590951

### LINKED ISSUE #672: Fix windows fetching
https://github.com/kblok/puppeteer/pull/12/commits/7f00860abd3c1efbca21351a8bdb12422dbe7aa2

### LINKED ISSUE #682: feat(chromium): roll Chromium to r590951
https://github.com/kblok/puppeteer/commit/a582acd682812bda2863e25196b7c3f6cedac2e1#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

### PR DESCRIPTION
**Pull request by Meir017**
closes #672
closes #682

### CODE REVIEW

File: lib/PuppeteerSharp/BrowserFetcher.cs [hunk 1]
```diff
@@ -286,6 +283,29 @@ private int GetRevisionFromPath(string folderName)
             return revision;
         }
 
+        private static string GetArchiveName(Platform platform, int revision)
+        {
+            if (platform == Platform.Linux)
```
Comment (kblok) [inline lib/PuppeteerSharp/BrowserFetcher.cs]: How about a `switch (platform)`?

Comment (Meir017) [reply lib/PuppeteerSharp/BrowserFetcher.cs]: Puppeteer also uses if's

Comment (kblok) [reply lib/PuppeteerSharp/BrowserFetcher.cs]: They don't have enums :joy:

**Comment by Meir017** [comment]
@kblok Done!

**Pull request merged**